src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/document-[document]/delete.svelte

```diff
@@ -37,16 +48,72 @@
             trackError(error, Submit.DocumentDelete);
         }
     };
+
+    enum Deletion {
+        'setNull' = 'Set document ID as NULL in all related documents',
+        'cascade' = 'All related documents will be deleted',
+        'restrict' = 'Document can not be deleted'
+    }
+
+    $: relAttributes = $attributes?.filter((attribute) =>
+        isRelationship(attribute)
+    ) as Models.AttributeRelationship[];
 </script>
 
-<Modal warning={true} onSubmit={handleDelete} bind:show={showDelete}>
+<Modal
+    icon="exclamation"
+    state="warning"
+    onSubmit={handleDelete}
+    bind:show={showDelete}
+    headerDivider={false}>
     <svelte:fragment slot="header">Delete Document</svelte:fragment>
 
     <p data-private>
-        Are you sure you want to delete <b>the document from {$collection.name}</b>?
+        Are you sure you want to delete <b
+            >the document from <span data-private>{$collection.name}</span></b
+        >?
     </p>
+
+    {#if relAttributes?.length}
+        <TableScroll>
+            <TableHeader>
+                <TableCellHead width={50}>Relation</TableCellHead>
+                <TableCellHead width={50}>Setting</TableCellHead>
+                <TableCellHead width={200} />
+            </TableHeader>
+            <TableBody>
+                {#each relAttributes as attr}
+                    <TableRow>
+                        <TableCell title="relation">
+                            <span class="u-flex u-cross-center u-gap-8">
+                                {#if attr?.twoWay}
+                                    <span class="icon-switch-horizontal" />
+                                {:else}
+                                    <span class="icon-arrow-sm-right" />
+                                {/if}
+                                <span data-private>{attr?.key}</span>
+                            </span>
+                        </TableCell>
+                        <TableCellText title="Settings">
+                            {attr?.onDelete}
+                        </TableCellText>
+                        <TableCellText title="description">
+                            {Deletion[attr?.onDelete]}
+                        </TableCellText>
+                    </TableRow>
```

Comment:
why we check for null here? if it comes crom the API, we can be sure if its there or not 👍🏻